scripts/pack_sources/pack_to_tarball.js

```diff
@@ -2,12 +2,12 @@ const exec = require('./exec_async').execAsync;
 const os = require('os');
 
 const packageName = os.platform() === 'darwin'
-  ? 'process_engine_runtime_macos.tar.gz'
-  : 'process_engine_runtime_linux.tar.gz';
+  ? 'atlas_engine_runtime_macos.tar.gz'
+  : 'atlas_engine_runtime_linux.tar.gz';
 
 // TODO: For some reason, this causes a "tar: .: file changed as we read it" error, when Jenkins tries to pack the linux sources.
 // const tarballCommand = 'tar -czvf process_engine_runtime_macos.tar.gz --exclude=\'.git*\' --exclude=\'Jenkinsfile\' --exclude=\'Dockerfile\' --exclude=\'.npmignore\' .';
-const tarballCommand = `tar -czf ${packageName} bin bpmn config dist node_modules scripts sequelize src test .eslintignore .eslintrc LICENSE package-lock.json package.json README.md reinstall.sh swagger.html tsconfig.json`;
+const tarballCommand = `tar -czf ${packageName} bin config dist migrations node_modules scripts src test .eslintignore .eslintrc LICENSE package-lock.json package.json README.md reinstall.sh swagger.html tsconfig.json`;
```

Comment:
Wäre eigentlich klüger gewesen, wenn du mit der RuntimeDomain angefangen hättest. 
Da existiert auch bereits eine GH Actions Integration.

Wir wollen aus den Archiven alle Dateien und Ordner rauslassen, die nicht systemrelevant sind. Das macht die Images wesentlich kleiner und übersichtlicher.
 
https://github.com/atlas-engine/RuntimeDomain.Application/blob/develop/scripts/pack_sources/pack_to_tarball.js#L8

```suggestion
const tarballCommand = `tar -czf ${packageName} bin config dist node_modules scripts migrations LICENSE package-lock.json package.json README.md swagger.html`;
```